Tests/TuistLoaderTests/Utils/ManifestFilesLocatorTests.swift

```diff
@@ -478,6 +478,19 @@ final class ManifestFilesLocatorTests: TuistUnitTestCase {
         XCTAssertNil(configPath)
     }
 
+    func test_locateConfig_where_tuist_file_is_not_a_directory() async throws {
+        // Given
+        try await createFiles(["tuist"])
+
+        // Then
+        do {
+            let configPath = try await subject.locateConfig(at: try temporaryPath())
+            XCTAssertNil(configPath)
+        } catch {
+            XCTFail("Expected no error, but got \(error)")
+        }
```

Comment:
```suggestion
         let configPath = try await subject.locateConfig(at: try temporaryPath())
         XCTAssertNil(configPath)
```

No need to do `do-try` dance - the test will automatically fail if `locateConfig` throws